lib/stripe_mock/data.rb

```diff
@@ -351,15 +351,18 @@ def self.mock_subscription(params={})
           data: [{
             id: 'si_1AwFf62eZvKYlo2C9u6Dhf9',
             created: 1504035973,
-            metadata: {},
+            metadata: {
+              company_deleted: false
```

Comment:
Thoughts on adding some comments to denote what each of change in this PR is needed for?

Reply:
Added comments.